### ISSUE TITLE
Non-working example

### ISSUE THREAD

**Issue by ivan-kleshnin** [isaachinman/next-i18next#350]
I encounter a number of issues with `examples/simple` and this library.

1) `react` and `react-dom` dependencies of `examples/simple` are local. Not sure if it's a good idea for a public example.

2) Errors. `examples/simple $ yarn dev` results in:

```
TypeError: Object(...) is not a function
    at Module../components/Title.js (/Users/ivankleshnin/Sandboxes/xxx/examples/simple/.next/server/static/development/pages/index.js:220:125)
```

When I comment out `<Title/>`:

```
TypeError: Object(...) is not a function
    at Module../pages/_error.js (/Users/ivankleshnin/Sandboxes/xxx/examples/simple/.next/server/static/development/pages/_error.js:202:125)
```

3. I also tried to follow the steps from README with `create-next-app` starter. The result was quite buggy, I have to say :( I can record a video or make a sandbox but in a nutshell:

3.1 Some page refreshes end with untranslated content (like 1/3) and some are ok. It feels like really bad race conditions are in play.
3.2 SSR is not working at all (I see untranslated strings in the HTML and they translate on client only, except the above case when they don't translate at all).

So basically I wanted to use this repo, but I couldn't make it work in a reasonable amount of time.
If the author is willing to help – I'll gladly provide more info on the above.

**Comment by isaachinman** [comment]
To run the example locally, follow [these steps](https://github.com/isaachinman/next-i18next/blob/master/CONTRIBUTING.md#setup-local-dev-environment). If those steps are not working for you, please do let me know.

I don't know anything about `create-next-app`. If you feel you've found a bug in `next-i18next`, please provide a reproducible repository for me to debug.

**Comment by lavir** [comment]
@isaachinman even following [these steps](https://github.com/isaachinman/next-i18next/blob/master/CONTRIBUTING.md#setup-local-dev-environment) 
and then 
` yarn run-example`

I got an error:
```javascript
yarn run-example                                                                                                                                                                                                                                                                                                                 ✔  22.56 祥  10:40:35
yarn run v1.16.0
$ yarn build && cd examples/simple && yarn && yarn dev
$ yarn clean && yarn build:cjs && yarn build:es
$ rm -rf dist && mkdir dist
$ BABEL_ENV=cjs babel src --out-dir dist/commonjs
Browserslist: caniuse-lite is outdated. Please run next command `yarn upgrade caniuse-lite browserslist`
Successfully compiled 23 files with Babel.
$ BABEL_ENV=es babel src --out-dir dist/es
Successfully compiled 23 files with Babel.
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
success Already up-to-date.
$ node index.js

/public/next-i18next/dist/commonjs/middlewares/next-i18next-middleware.js:1
import _objectSpread from "@babel/runtime/helpers/objectSpread";
       ^^^^^^^^^^^^^

SyntaxError: Unexpected identifier
```

**Comment by isaachinman** [comment]
If I fresh clone the repo, then do `yarn && yarn build && yarn build:examples/simple`, and then do `yarn run-example`, I cannot reproduce this problem.

**Comment by lavir** [comment]
@isaachinman could you say what NodeJS version are you use? I tried running on two different Macs with NodeJS 12.4.0 with the same unsuccessful result.

**Comment by isaachinman** [comment]
This is not related to Node versioning. I can confirm it works fine in v12.4.0.

Pay attention to the output of your console. You've probably got `next-i18next` linked somewhere else and need to unlink it first.
